Configuration/Settings.yaml

```diff
@@ -1,3 +1,24 @@
+Flowpack:
+  ElasticSearch:
+    ContentRepositoryAdaptor:
+      driver:
+        version: '2.x'
```

Comment:
I prefere to let this more open, as Elastic can be strange in term of "breakingness" ... maybe at some point a more specific driver will be need, like 5.* and/or 5.5-5.6